src/julia.h

```diff
@@ -365,6 +365,9 @@ typedef struct _jl_method_t {
 // hidden fields:
     // lock for modifications to the method
     jl_mutex_t writelock;
+
+    // iddict: root -> index into `roots`
+    jl_genericmemory_t *roots_table;
```

Comment:
How is GC supposed to find this field? See `jl_method_type` in `jltypes.c`. GC only scans the fields that are part of the type.

Reply:
Didn't realize, thanks!